Library/Models/SurveyScript.cs

```diff
@@ -37,5 +37,10 @@ public class SurveyScript
         /// Warning messages that resulted from the parse
         /// </summary>
         public IEnumerable<string> WarningMessages { get; set; }
+
+        /// <summary>
+        /// Indicates if script with unfixed positions is allowed
+        /// </summary>
+        public bool UnfixedIsOk { get; set; }
```

Comment:
No new/updated unit tests for this?

Reply:
We added two integration test for this, Not sure what should I test in unit tests

Reply:
I was thinking about [these](https://github.com/NIPOSoftware/Nfield-SDK/blob/master/Tests/Services/NfieldSurveyScriptServiceTests.cs) but the api integration tests are way more useful of course